addons/delete-others/userscript.js

```diff
@@ -0,0 +1,87 @@
+export default async function ({ addon, console, msg }) {
+  const vm = await addon.tab.traps.vm;
+
+  const types = ["sound", "costume"];
+  let deletedItems = [];
+  let deleted;
+  let target;
+
+  function getRestoreFun(type) {
+    deletedItems.reverse();
+    if (type === "costume") {
+      deletedItems.forEach((costume) => {
+        vm.addCostume(costume.md5, costume, target.id, "");
+      });
+    } else if (type === "sound") {
+      deletedItems.forEach((sound) => {
+        vm.addSound(sound, target.id);
+      });
+    }
+    deletedItems = [];
+  }
+
+  addon.tab.createEditorContextMenu(
+    async (ctx) => {
+      const deleteOthersStr = msg("deleteOthers");
+      if (
+        await addon.tab.confirm(
+          deleteOthersStr.charAt(0).toUpperCase() + deleteOthersStr.slice(1),
```

Comment:
I think this should be a separate string. In English, it should be "Delete Others", not "Delete others".

Reply:
Fair